tests/utils/output_verification.hpp

```diff
@@ -30,37 +30,40 @@
 #include <limits>
 #include <cmath>
 #include <complex>
+#include <string>  
+#include <iostream> 
+#include <fstream> 
+#include <sstream> 
 
 #include <assert.h>
 
 
 
 /**
- * Attempts to read in a double value from a given file into a given memory
+ * Attempts to read in a value from a given file into a given memory
  * location.
+ * 
+ * @tparam T 		The datatype of the value
  *
- * @param[in]  in  The input file
- * @param[out] out Where to store the read value.
+ * @param[in]  in  	The input file
+ * @param[out] out 	Where to store the read value.
```

Comment:
if these are tabs (after in and out), then they should be removed or replaced with spaces